### PR TITLE
test(refactor): periphery tests to use Bulloak

### PR DESCRIPTION
**Pull request by smol-ninja**
Closes https://github.com/sablier-labs/v2-core/issues/799

## Tasks
- [x] Refactor periphery tests to comply to bulloak check
- [x] Rewrite periphery tests to conform to BTT specs

### CODE REVIEW

File: test/periphery/integration/merkle-campaign/factory/create-merkle-instant/createMerkleInstant.tree [hunk 1]
```diff
@@ -1,9 +1,9 @@
-createMerkleInstant.t.sol
-├── when the campaign name is too long
+CreateMerkleInstant_Integration_Test
+├── when campaign name exceeds 32 bytes
```
Comment (andreivladbrg) [inline test/periphery/integration/merkle-campaign/factory/create-merkle-instant/createMerkleInstant.tree:2]: short form `when name too long ` ?

File: test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree [hunk 2]
```diff
@@ -1,9 +1,9 @@
-createMerkleLL.t.sol
-├── when the campaign name is too long
+CreateMerkleLL_Integration_Test
+├── when campaign name exceeds 32 bytes
```
Comment (andreivladbrg) [inline test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree:2]: same as above

File: test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 │  └── it should revert
-└── when the campaign name is not too long
-   ├── given the campaign has been created already
+└── when campaign name not exceeds 32 bytes
+   ├── given campaign already exists
    │  └── it should revert
-   └── given the campaign has not been created already
+   └── given campaign does not exist
```
Comment (andreivladbrg) [inline test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree:7]: we can remove `does`

File: test/periphery/integration/merkle-campaign/factory/create-merkle-lt/createMerkleLT.tree [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+CreateMerkleLT_Integration_Test
+├── when campaign name exceeds 32 bytes
 │  └── it should revert
-└── when the campaign name is not too long
-   ├── given the campaign has been created already
+└── when campaign name not exceeds 32 bytes
+   ├── given campaign already exists
    │  └── it should revert
-   └── given the campaign has not been created already
+   └── given campaign does not exist
```
Comment (andreivladbrg) [inline test/periphery/integration/merkle-campaign/factory/create-merkle-lt/createMerkleLT.tree:7]: same

**Review by andreivladbrg — COMMENTED** [review]
Generally looks good, few comments bellow:

File: test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree (hunk 3 above)
Comment (smol-ninja) [reply test/periphery/integration/merkle-campaign/factory/create-merkle-ll/createMerkleLL.tree]: Agree

File: test/periphery/integration/merkle-campaign/factory/create-merkle-instant/createMerkleInstant.tree (hunk 1 above)
Comment (smol-ninja) [reply test/periphery/integration/merkle-campaign/factory/create-merkle-instant/createMerkleInstant.tree:2]: Agree

Comment (andreivladbrg) [reply test/periphery/integration/merkle-campaign/factory/create-merkle-instant/createMerkleInstant.tree]: 3dc5e0ca68fd445cb12f7db3a4becfa4ae5c701a

**Review by andreivladbrg — APPROVED** [review]
I have pushed 2 commits.

Now it looks good on my end

**Comment by smol-ninja** [comment]
Great. Thanks for making the changes. Looks good to me as well.

**Pull request merged**